cockatrice/src/gameselector.h

```diff
@@ -23,6 +23,7 @@ private slots:
     void actCreate();
     void actJoin();
     void checkResponse(const Response &response);
+    void updateButtonChoices(const QModelIndex &);
```

Comment:
I feel this was merged too soon. One of the reviewers @Daenyth hasnt replied yet.